<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBoxProfile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Profiles</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <widget class="QLabel" name="labelProfile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Profile</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="labelProfileGameValue">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="labelProfileSIDValue">
         <property name="text">
          <string notr="true">54321</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="labelProfileGame">
         <property name="text">
          <string>Game</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labelProfileSID">
         <property name="text">
          <string>SID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelProfileTID">
         <property name="text">
          <string>TID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonProfileManager">
         <property name="text">
          <string>Manager</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxProfiles"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="labelProfileTIDValue">
         <property name="text">
          <string notr="true">12345</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBoxStationaryRNGInfo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>RNG Info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonGenerate">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBoxStationarySettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_14"/>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBoxStationaryFilter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Filters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0" rowspan="5">
        <widget class="IVFilter" name="ivFilter" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelNature">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Nature</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="CheckList" name="comboBoxNature"/>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButtonAnyNature">
         <property name="text">
          <string>Any</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelHiddenPower">
         <property name="text">
          <string>Hidden Power</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="CheckList" name="comboBoxHiddenPower"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButtonAnyHiddenPower">
         <property name="text">
          <string>Any</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelAbility">
         <property name="text">
          <string>Ability</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboBoxAbility">
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">H</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelGender">
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="comboBoxGender">
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">♂</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">♀</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="labelShiny">
         <property name="text">
          <string>Shiny</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="checkBoxStationaryShiny">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="TableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>60</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuStyle">
     <property name="title">
      <string>Style</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionLight"/>
    </widget>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionGerman"/>
     <addaction name="actionEnglish"/>
     <addaction name="actionSpanish"/>
     <addaction name="actionFrench"/>
     <addaction name="actionItalian"/>
     <addaction name="actionJapanese"/>
     <addaction name="actionKorean"/>
     <addaction name="actionChinese"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="menuStyle"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionGerman">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="actionItalian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Italian</string>
   </property>
  </action>
  <action name="actionJapanese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Japanese</string>
   </property>
  </action>
  <action name="actionKorean">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Korean</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>IVFilter</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/IVFilter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CheckList</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/CheckList.hpp</header>
   <slots>
    <slot>resetChecks()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAnyNature</sender>
   <signal>clicked()</signal>
   <receiver>comboBoxNature</receiver>
   <slot>resetChecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1081</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>1020</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAnyHiddenPower</sender>
   <signal>clicked()</signal>
   <receiver>comboBoxHiddenPower</receiver>
   <slot>resetChecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1095</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>1000</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
